admin/includes/languages/english/lang.invoice.php

```diff
@@ -11,6 +11,7 @@
     'ENTRY_PAYMENT_METHOD' => 'Payment Method:',
     'ENTRY_SHIPPING' => 'Shipping:',
     'TEXT_INFO_ATTRIBUTE_FREE' => '&nbsp;-&nbsp;FREE',
+    'INVOICE_ERROR_ORDER_DOES_NOT_EXIST' => 'No Information for invoice ',
```

Comment:
Instead of creating two new constants, please move `ERROR_ORDER_DOES_NOT_EXIST` from `./includes/languages/english/lang.orders.php` to `includes/languages/lang.english.php` and just use that.